GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -640,7 +640,7 @@ public Result MultiPackIndexExpire(string gitObjectDirectory)
 
         public Result MultiPackIndexRepack(string gitObjectDirectory, string batchSize)
         {
-            return this.InvokeGitAgainstDotGitFolder($"-c pack.threads=1 multi-pack-index repack --object-dir=\"{gitObjectDirectory}\" --batch-size={batchSize}");
+            return this.InvokeGitAgainstDotGitFolder($"-c pack.threads=1 multi-pack-index repack --object-dir=\"{gitObjectDirectory}\" --batch-size={batchSize} --no-progress");

```

Comment:
@derrickstolee do you think I should pass `--no-progress` in for `write` and `expire` as well?  I was a bit on the fence about it as those actions don't have any progress indicators.

Reply:
As long as the Git command still succeed, this is a good future-proofing.

Reply:
That sounds good to me, thanks!